public/stylesheets/style.css

```diff
@@ -264,11 +265,15 @@ body>header img {
 }
 
 #favorites-icon a {
-  background-image: url('/images/icon-heart-filled.svg');
+    -webkit-mask-image: url('/images/icon-heart-filled.svg');
+    mask-image: url('/images/icon-heart-filled.svg');
```

Comment:
`mask-image` is niet ondersteund in Internet Explorer (https://caniuse.com/#feat=mdn-css_properties_mask-image).......

Maar ik weet niet of we daar een moer om geven 🤷